slack/table_slack_conversation_members.go

```diff
@@ -0,0 +1,61 @@
+package slack
+
+import (
+	"context"
+
+	"github.com/slack-go/slack"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableSlackConversationMembers() *plugin.Table {
+	return &plugin.Table{
+		Name:        "slack_conversation_members",
+		Description: "Retrieve members of a conversation.",
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.SingleColumn("channel"),
+			Hydrate:    listConversationMembers,
+		},
+		Columns: []*plugin.Column{
+			{Name: "channel", Type: proto.ColumnType_STRING, Description: "ID of the conversation to retrieve members for."},
```

Comment:
```suggestion
			{Name: "conversation_id", Type: proto.ColumnType_STRING, Transform: transform.FromField("Channel"), Description: "ID of the conversation to retrieve members for."},
```

Even though the request parameter is called `channel`, I think changing this column to `conversation_id` is more consistent with our other column names, e.g., `id` in `slack_conversation`, and including the word `conversation` in it helps differentiate this column from the one containing member IDs.